manifests/docker.pp

```diff
@@ -42,7 +46,8 @@
   }
   ~>
   service { 'docker-xylem':
-    ensure => running,
+    ensure  => running,
+    require => File['/run/docker/plugins'],
```

Comment:
Although now that I think about it, the directory declaration really should be in an `unless defined() {...}` block in case something else manages it.